requirements/dev.in

```diff
@@ -6,6 +6,6 @@ edx-i18n-tools            # For i18n_tool dummy
 pip-tools                 # Requirements file management
 tox                       # virtualenv management for tests
 tox-battery==0.5.1        # Makes tox aware of requirements file changes (it's experimental, so keep it pinned)
-twine                     # Utility for PyPI package uploads
+twine==1.11.0             # Utility for PyPI package uploads
```

Comment:
@nasthagiri: Had to pin twine to deal with the bleach issue.

Reply:
With auto-deployment on new tags configured in Travis CI, do we still need twine in the development dependencies?  I think this was primarily for manual upload of new releases.

Reply:
FYI: I created ENT-1293 for this clean-up.  I have many upgrades of edx-drf-extensions to get out that are blocked.